dmv/test/dmv_test.js

```diff
@@ -1,55 +1,58 @@
-var DMV = require('../dmv')
```

Comment:
Well done submitting consistently formatted code! 

However when completing code to pass assigned tests, you don't want to include changes to the tests themselves - that is usually done as a separate Issue & Pull Request.